.changeset/wicked-brooms-refuse.md

```diff
@@ -0,0 +1,10 @@
+---
+"@ledgerhq/types-live": patch
+"ledger-live-desktop": patch
+"live-mobile": patch
+"@ledgerhq/live-common": patch
+"@ledgerhq/live-cli": patch
+"@ledgerhq/live-env": patch
```

Comment:
Shouldn't this all be `minor`?
Seem more relevant according to [semver](https://semver.org/) since this is more akin to a new feature or feature update than a bug fix

Reply:
You may be right 🤷🏼